src/Console/Shell.php

```diff
@@ -242,12 +249,14 @@ public function startup()
      */
     protected function _welcome()
     {
+        $phpInfo = explode('~', phpversion());
+        $this->phpVersion = (isset($phpInfo[0]) ? $phpInfo[0] : phpversion());
```

Comment:
The consensus was to fix it the proper way by only displaying it alltogether for the mentioned main page and --version --help things. 
This here only fixes a small symptom, so let's please focus on the actual way to solve it right.